.travis.yml

```diff
@@ -15,15 +15,18 @@ before_install:
+  - secure: wYlC9mi41h5geyHiqoTF1Itn9HHfuHf0uD3saCgxz2OORbJ35ri5elQ75bBD6oD4k/K49gIpIs+bjNL3Svrr8ECWu0RfxqfkkhtYkrMZzmyVl2njidoJPmW1eEsQqeHKOcqMUfB8J50YIH5LsN/JsGy/6AORqn/+gmgXBMD+bqfXdzP68zRlnkoS1ayX1YsEzRiXmYAGRW48N3akGqmZVW/t3V4b0p7CJ89Lfa/RfLIju0eXOc2En/mVRGyRZIJ4Z0LrtBg7lED5l9fUwztnCPItdc1+vWm5Dt/LGx60xWfcwbPN9kYncW9ANAuIahTc0Y6ajohgSr+9O7zjssGGxxPigbtFaqIF8Lvg+pP149Nm2UjiCT0yUbtv4U5IvfCP/v8LKGOqHYi95QV3jdqHqKgsCVwU9nDPVCIQjOqT6bZtbwjfN9ugvaahyFowXBh259GpXjHtwNG4bbWtfZB4lWqTrwYX739MazRDe0OQOaolQq4O4fM26gd+gjxX6twvF3ImK7+OlaKWY1QgJOeCJPcZiCRH4tcjemZaoHZdwu//njjiUO0rV3vTFpEfMxXkakPoroW1XXFJQgp9kUoe1/i7lg1Kzdw48c7X7jAcT8gqo6B5ocN3+qLXwhgV5dcsyHf9rfKFd5k/L9oQJw8NEAJ6m76td4OLynsvcXgkWrc=
+  - secure: TvN3ZsmYx8pDVqD5EKr7126N4b4H+4VX6lStuEyAVsZo72Ekg4bY7pjOzNEVbOjoB8nQgd6I3Xc3Duc5FurekFefkbBRSaEF4XU2nmD02Smdm83xjk5eEObFdqXKC2Ovqh4BcFG7sNToJd4OGaP2KtSnJGEAcHJkFVbq81o92qmf2ZqxNvD8DDicQaXwIk9Xx1uAFfpBPX8v01DNFueX3LCd72z26BC71dv8kEW1Ftuz9cDuRtWMjDfYZHwxe+HCrf7P3Y5MyrWeLer1yW82G5k6By80gg9ii2tOGPYfmueyxiz1Db6oXym0ZgP0Zq4qBe7vG88EPCSFuVZMhGISZLLNYowbLm5k/rTndcl/U9FQzEN1E0aI+xkBRdr0yYIpr0t/rrFCHsrrmlWKHojPudBbLvRnMcVEsZzbLElln3LaqNVuM9ap8wOKIo1yUNSE5aflFQV4RHP2P7ipHHrx+o8ygfQqQeNFo29XTiUEFEStoofU2jYtQG1DROra1ayGFAYk12WO+Rx+NkhHxlpedhmqIN/odrqQiYcJIbXCEgaJHAP+KfAMUX3wIj5AD5P4Yqf71DINbMIAm97pXClLQmTvttFlI1TpYN8c81HlEsPktoBesGHpBrqmr32CKn0erbDdLS7rPQsGXa17lxci5daQ91g3UBvQDvQBQpjjTaU=
+  - secure: GpnYIe/Ba0+2I0gCvq0V6SBk8f5yW1enmyy9BglwoXgHKBU78dXCnNpfyqnChN5x267JTbPuQbiaZfrms/pPQZxItb40sBgKotxIaTBeLKwMXnRJxKlD8fmHraHs38ioSY21z3I810a/Kd6Jz3L6L4vH2wnscWQBFEzjLpfuYys1o7WfppuuwEYJPnh8MdeyTmyfWMd0klumDcDHc5HyDqVnemterY0W8HqrbX+ydLfXUQpI7GiBV/rDHiXy7LDcYNOB2tLEOwmb9Ex1QoNX6n6hN2u0aCk14pflGEQYdvosNt7AjzQK3ek/S/ipXcXWWyNwD18m7zJ+FQg8+IJEZXWI3CjXKa2PHbTAv4Gz30jv+T8evldF7Iyc5phApdEBsMdE54wLDOtEV05rDo0pSz+cTimTaW/bDXmMcVQ3wzGmdpZg405rfriYBKL97CvxyCxHhfyDhTlmSj56rJ1Is4d1i8xoN8hhNJx51MVuwgRWTS9FrjAjogkfdVqCs015A7JIWopOcxHt5PYCvtA4LYQAcgyycQ0qPf7MPYbO4QaoD6VTGysagB6wMpydXEMDkIyKZEVZrlxByzl2UomK7ahosxB0O5w06g5RF2j03VOmyKUhskzJawEwyZ2wZS/VdFlV7KUnOOfzCLY9iTdlxkXyghOjgFKp2LaCrDqLOPc=
+  - secure: aNUiOhCdJk+S2TEay6FCJxJHdI/JAOeb7fNXHtVKxZ6JW6fYw2tdGJIMOBUnHKimIOjZSM03yr9dZfPvYzYxwirWNbXpuTyCNxj/g6NSrkc99dyznP9tDq16Bs+1DxLx7K3ns3M2AvKrX726pr/LsujbnrIclVcdwXEuUk4ThWFEbeMSKZlHrDMpzpC17NK8bVsJEAxUh42Dm4tmHDf2Kr+s3GIrGeGafQRGsv6aRz/Tup8Ms8W9eqPoGYgcjHWCBpTeuHy4gg4ci6zkSj5PiudAAMZN/1rx3mNr0tOXFQ5oKI+kLOQ/f2RDmuw/s6wW1DkTQMj6+2gTrlqmKwa01hxYGQNI8YmpMb5URCMgRKir1QNytv+XFM8A+MTnVY90hBiXc52c+vaUa1W3BiGM8vYQUC+ylPXVXjsCjYhTAYUNY8uZfu/LfjoVGCEXhGYNhdjdM5e416BGC+CH9/u8oNb6eIRoijWjVC3/uKLm2JoRUe5rQmR6fs9TlRKKPm3KK8FAcoBb+H5fIEv+4L6r8AQyDZaNEpq4ocQnGB/mumWS0LkvnwlIguFrctY/tAgwzBxWR+MUCxdowDWQ1qe0W8/gsrCrT8Ni1sV4/uaSYGotHwe0iDBXmZ0pQNS7OGc0iunBXL1kZrTddTX5k5W6kjvoN/i+yy7L/lfU8OSK2kU=
```

Comment:
Why do we need so many encrypted keys?

I would recommend maybe to define them directly in travis website, you know at least the name of the variable.

Example

![image](https://cloud.githubusercontent.com/assets/1110551/19379403/188dcc58-91f2-11e6-90b3-5e558cb76bb5.png)